aws-serverless-java-container-struts2/src/main/java/com/amazonaws/serverless/proxy/struts2/Struts2LambdaHandler.java

```diff
@@ -0,0 +1,45 @@
+package com.amazonaws.serverless.proxy.struts2;
```

Comment:
I'm not a struts expert. Does this mean that Struts automatically detects all "resources" in the package and therefore we can provide a default handler class? To make this clearer, we should include a [quick start guide](https://github.com/awslabs/aws-serverless-java-container/wiki#quick-starts) in the wiki and [an archetype](https://github.com/awslabs/aws-serverless-java-container/tree/master/archetypes) that sets up the SAM template automatically.

Reply:
I add a quick start guide, an archetype and the pet store sample for Struts2 based lambdas.